### PR TITLE
New Account Switcher

### PR DESCRIPTION
**Pull request by harpreet9629**
Here's Finally I changed and fixed it!

Screenshot_1
Screenshot_2
Screenshot_3

### CODE REVIEW

File: src/app/change-account-selector/change-account-selector.component.html [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
   #changeAccountSelectorRoot
   id="changeAccountSelectorRoot"
 >
   <!-- Selector (unopened state) -->
-  <div class="d-flex align-items-center justify-content-between pl-10px pr-10px change-account-selector__hover">
-    <div *ngIf="globalVars.loggedInUser" class="change-account-selector__ellipsis-restriction cursor-pointer">
+  <div class="d-flex align-items-center justify-content-between pl-5px pr-5px">
+    <div *ngIf="globalVars.loggedInUser" class="change-account-selector__ellipsis-restriction d-flex flex-row align-items-center cursor-pointer">
+      <div class="feed-create-post__avatar align-items-center change-account-selector__account-image" [avatar]="globalVars.loggedInUser.PublicKeyBase58Check"></div>
+      <div class="change-account-selector__acount-name">
```
Comment (lazynina) [inline src/app/change-account-selector/change-account-selector.component.html:11]:
```suggestion
      <div class="change-account-selector__acount-name change-account-selector__ellipsis-restriction">
```

when a long name is selected, it currently doesn't show ellipsis. this should resolve that.

**Review by lazynina — CHANGES_REQUESTED** [review]
other than this, it looks good to me. Would like to get @maebeam's eyes for styling of this as well before we merge

**Comment by harpreet9629** [comment]
Ok i am waiting, but most of the peoples likes it

**Review by maebeam — APPROVED** [review]

**Review by lazynina — APPROVED** [review]

**Comment by harpreet9629** [comment]
Thank You Guys :)

**Pull request merged**